addons/resource.language.en_gb/resources/strings.po

```diff
@@ -24049,7 +24049,19 @@ msgctxt "#39201"
 msgid "HDR10+"
 msgstr ""
 
-#empty strings from id 39202 to 39999
+#. Subtitle line spacing setting
+#: system/settings/settings.xml
+msgctxt "#39202"
+msgid "Line spacing"
+msgstr ""
+
+#. Subtitle line spacing setting help
+#: system/settings/settings.xml
+msgctxt "#39203"
+msgid "Setting the spacing too low may result in overlapping when the \"Background type\" is set to \"Box\"."
```

Comment:
add missing setting explanation,
imo better specify a more general explanation of the problem, such as:
```suggestion
msgid "Set the amount of space between lines of text. A too low value may cause the boxes to overlap when using \"Background type\" setting."
```